packages/gatsby-source-shopify/__tests__/fixtures/shopify-nodes/ShopifyCollection.json

```diff
@@ -1 +1 @@
-[{"description":"Test Collection Description","descriptionHtml":"Test Collection Description","feedback":{"details":[],"summary":"This collection couldn’t be published to Google"},"handle":"test-collection","image":null,"legacyResourceId":"278407151822","productsCount":2,"ruleSet":{"appliedDisjunctively":false,"rules":[{"column":"IS_PRICE_REDUCED","condition":"","relation":"IS_NOT_SET"}]},"seo":{"description":null,"title":null},"sortOrder":"BEST_SELLING","storefrontId":"Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI3ODQwNzE1MTgyMg==","templateSuffix":"","title":"Test Collection","updatedAt":"2022-03-14T22:39:00Z","shopifyId":"gid://shopify/Collection/278407151822","products___NODE":["87b524a6-a707-5eb9-b193-059b3fcf96e5","1386d0f3-354b-584c-a6bb-d11936c8f6fb"],"metafields___NODE":[],"id":"5e2d818c-9edd-5d9a-9b8b-a87b59518efc","internal":{"type":"ShopifyCollection","contentDigest":"097e80a6bd493fd407351894b026070f","owner":"gatsby-source-shopify","counter":47},"children":[],"parent":null}]
\ No newline at end of file
+[{"description":"Test Collection Description","descriptionHtml":"Test Collection Description","feedback":{"details":[],"summary":"This collection couldn’t be published to Google"},"handle":"test-collection","image":null,"legacyResourceId":"278407151822","productsCount":{"count":2,"precision":"EXACT"},"ruleSet":{"appliedDisjunctively":false,"rules":[{"column":"IS_PRICE_REDUCED","condition":"","relation":"IS_NOT_SET"}]},"seo":{"description":null,"title":null},"sortOrder":"BEST_SELLING","storefrontId":"Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzI3ODQwNzE1MTgyMg==","templateSuffix":"","title":"Test Collection","updatedAt":"2022-03-14T22:39:00Z","shopifyId":"gid://shopify/Collection/278407151822","products___NODE":["87b524a6-a707-5eb9-b193-059b3fcf96e5","1386d0f3-354b-584c-a6bb-d11936c8f6fb"],"metafields___NODE":[],"id":"5e2d818c-9edd-5d9a-9b8b-a87b59518efc","internal":{"type":"ShopifyCollection","contentDigest":"097e80a6bd493fd407351894b026070f","owner":"gatsby-source-shopify","counter":47},"children":[],"parent":null}]
```

Comment:
Do we want to prettify this file as well? (see `packages/gatsby-source-shopify/__tests__/fixtures/shopify-nodes/ShopifyInventoryLevel.json`)

Reply:
Addressed in https://github.com/gatsbyjs/gatsby/pull/39082/commits/2feb7371e217fcb915581ebf528b26b7ac43e12d